source/coll_db.f90

```diff
@@ -484,19 +591,52 @@ subroutine cdb_writeFam
 
   integer j, famUnit
 
-  call f_requestUnit("coll_families.dat", famUnit)
-  call f_open(unit=famUnit,file="coll_families.dat",formatted=.true.,mode="w",status="replace")
+  call f_requestUnit("coll_families_dump.dat", famUnit)
```

Comment:
don't you want to have the name of the file in a variable in the module header?

Reply:
I don't really see the value in a moule-wide variable used only one place? I can make it a parameter in the subroutine, but that too seems a bit unnecessary.